google-cloud-storage/src/test/scala/docs/scaladsl/GCStorageSourceSpec.scala

```diff
@@ -221,23 +257,30 @@ class GCStorageSourceSpec
 
     "fail with error when bucket listing fails" in {
 
-      mockBucketListingFailure()
+      mock.simulate(
+        mockTokenApi,
+        mockBucketListingFailure()
+      )
 
       val listSource = GCStorage.listBucket(bucketName, None)
 
       listSource.runWith(Sink.seq).failed.futureValue.getMessage shouldBe "[400] Bucket listing failed"
     }
 
-    "return empty source listing bucket with wrong settings" in {
+    // TODO Is this really the desired behavior?
```

Comment:
Couldn't get this test to pass (it expects a silent failure) and not sure if it should anyway. Other tests with `mockBucketListingFailure` like the one immediately above it fail loudly.

Reply:
It seems like an odd way to handle a misconfiguration (if that configuration is really necessary in the first place). I think it's fine to drop this.